src/Input/Input.tsx

```diff
@@ -89,29 +89,42 @@ export interface InputProps extends InputHTMLAttributes<HTMLInputElement> {
    * @deprecated
    */
   static?: boolean;
+  /** Quando attivo rimuove il componente contenitore dell'Input. Utile per un controllo maggiore dello styling */
+  noWrapper?: boolean;
+  /** Funzione callback per quando il campo riceve il focus */
+  onFocus?: React.FocusEventHandler<HTMLInputElement>;
+  /** Funzione callback per quando il campo perde il focus */
+  onBlur?: React.FocusEventHandler<HTMLInputElement>;
 }
 
 type InputState = { isFocused: boolean; hidden: boolean; icon: boolean };
 
 export class Input extends React.Component<InputProps, InputState> {
+  public static defaultProps = {
+    onFocus: noop,
+    onBlur: noop
+  };
```

Comment:
No need to define default values here.

Reply:
right, we are already checking if they are defined, my bad